### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by thorstenwagner** [BiAPoL/napari-clusters-plotter#241] [General]
Hi,

is it possible to make another pre-release that contains the histogram PR etc? :-)

Best,
Thorsten

**Comment by lazigu** [comment]
Hi @thorstenwagner, 
Yes, we should definitely make a new release. Sorry for the delay! I changed the job in April because I started a PhD position in another group, and due to some other incoming changes I haven't made one before I left (I was hoping to get 1D histogram with span selector merged first @zoccoler). @haesleinhuepf, could I ask you to make the new release since you know better whether the plugin is used upcoming days in any courses, and I'm not sure if you want currently opened PRs to be in the next release?

**Reply by zoccoler** [reply]
Hi @lazigu , that PR should be ready today, I am updating it with the necessary changes.

**Reply by lazigu** [reply]
No rush from my side, just letting everybody know what was going on through my head and why I didn't make a release before I left :) Was also hoping that more people from the group play around with the unreleased version and really make sure there are no bugs, since it's bringing so many changes.

**Reply by haesleinhuepf** [reply]
I'm happy  to release the release; potentially by the end of the week :-)

**Reply by zoccoler** [reply]
Maybe we could use Milestones to check what is needed for a next release. :)
I tentatively created one [here](https://github.com/BiAPoL/napari-clusters-plotter/milestones). 
It is easy to use! We just have to tag it in issues or PR we think should be closed before making the release, what do you think?
I tagged #237 as an example.
Otherwise, we just delete the milestone.

**Comment by thorstenwagner** [comment]
Congrats @lazigu to your new position :-)  Looking forward to the release :-)

**Comment by thorstenwagner** [comment]
Done! Thanks!
